src/app/portlets/dot-edit-page/content/services/dot-page-state/dot-page-state.service.ts

```diff
@@ -22,6 +22,7 @@ import { DotRouterService } from '@services/dot-router/dot-router.service';
 @Injectable()
 export class DotPageStateService {
     state$: Subject<DotRenderedPageState> = new Subject<DotRenderedPageState>();
+    contentAdded$ = new Subject<DotRenderedPageState>();
```

Comment:
If you already have a place where you can tell if content were added or not, why do you need another subject here?